pandas/core/window/online.py

```diff
@@ -52,7 +52,7 @@ def online_ewma(
         exponentially weighted mean accounting minimum periods.
         """
         result = np.empty(values.shape)
-        weighted_avg = values[0]
+        weighted_avg = values[0].copy()
```

Comment:
I moved the copy here inside the function, instead of copying the full array before passing it to the function, since here we only need to copy the first row of the array.